docs/core/integration-quality-scale/rules/iqs013.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "IQS013 - Entities have translated names"
+---
+
+IQS013 is as follows:
+
+Entities have translated names
+
+## Reasoning
+
+Home Assistant is used by people all over the world.
+To also make it easier for non-English speakers to use Home Assistant, it is important that entities have translated names.
+This makes it easier for people to understand what the entity is.
```

Comment:
I somehow feel that we also need to explain that if there are values that come from the API set by the user, that we allow that name not to be translated. (Think of a user setting a lock to the name "front door lock" in the app). But that when we have an integration that provides a lot of data points (lg_thinq, home_connect, smartthings, matter), we rather want the developers to redefine the entities again in the code, so we can add more context like device classes, translations and so forth.

Reply:
I'm not following what the entity name has to do with the other properties you mention in the comment.

Reply:
Oh, you're talking about the default translation of entity names with a device class set.

Reply:
Just add one sentence to the info paragraph below, saying that if there are more than one entity per device class in a platform, the platform should use translation keys for the entity name instead of relying on the default names coming from the device class.

Reply:
I mean more like, if the api returns a user defined name for a property, we should not try to translate it. But now that I think of it again, I feel like that this only happens on device level and hardly on individual entities